probatus/feature_elimination/feature_elimination.py

```diff
@@ -72,48 +80,37 @@ def __init__(self, clf, search_space, search_schema='random', step=1, min_featur
                 elimination. The recommended model is LightGBM, because it by default handles the missing values and
                 categorical variables.
 
-            search_space (dict of sklearn.ParamGrid): Parameter search space, which will be explored during the
-                hyperparameter search. In case `grid` search_schema, it is passed to GridSearchCV as `param_grid`, in case
-                of `random` search_schema, then this value is passed to RandomSearchCV as `param_distributions` parameter.
+            step (Optional, int or float): Number of lowest importance features removed each round. If it
+                 is an int, then each round such number of features is discarded. If float, such percentage of remaining
+                 features (rounded down) is removed each iteration. It is recommended to use float, since it is faster
+                 for a large number of features, and slows down and becomes more precise towards less features. Note:
+                 the last round may remove fewer features in order to reach min_features_to_select.
 
-            search_schema (Optional, str): The hyperparameter search algorithm that should be used to optimize the model.
-                It can be one of the following:
+            min_features_to_select (Optional, unt): Minimum number of features to be kept. This is a stopping criterion
+                of the feature elimination. By default the process stops when one feature is left.
 
-                - `random`: [RandomSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RandomizedSearchCV.html)
-                  which randomly selects hyperparameters from the prowided param_grid, and performs optimization using
-                  Cross-Validation. It is recommended option, when you optimize a large number of hyperparameters.
-                - `grid`: [GridSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.GridSearchCV.html)
-                  which searches through all permutations of hyperparameters values from param_grid, and performs
-                  optimization using Cross-Validation. It is recommended option, for low number of hyperparameters.
+            cv (Optional, int, cross-validation generator or an iterable): Determines the cross-validation splitting
```

Comment:
You mention 'cross validation generator' but if I understand correctly, you got smart and also allow for 'cross validation estimators'. See https://scikit-learn.org/stable/glossary.html#term-cv-splitter for terminology. I recommend using the same terms & mentioning here that both the CV generators and estimators are allowed

Reply:
In this case not. It supports some specific cross validation estimators (GridsearchCV and RandomsearchCV) as clf parameter,  but not as cv parameter.

Indeed, I added them to the clf parameter description. Nice catch!